config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  resources :errors, only: [:show], constraints: { id: /\d{3}/ }
```

Comment:
So if that match is correct, I was wrong in telling you to add this constraint. You can safely remove this `constraints` key, sorry.